rfcs/000-resource-notices.md

```diff
@@ -0,0 +1,88 @@
+- Feature Name: Resource Notives
+- Start Date: 2021-03-22
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Resource Notices
+
+## Summary
+
+Resource Notices allows Amundsen administrators to show informational, warning and alert messages related to the different resources (tables, dashboards, people) we expose in Amundsen
+
+## Motivation
+
+This feature will help administrators to show messages related to deprecation, updates (or lack of), and informational messages that are related to specific resources. In Lyft's case, we are using them to point final users to a improved versions of current surfaced tables, so final users employ more trusted data.
+
+## Guide-level Explanation (aka Product Details)
+
+An Amundser final user can see a notice in the table/dashboard/user detail page whenever his system admin adds one related to it.
+
+A notice is a small box with an icon and a message that can contain HTML markup (like links and bolded text).
+
+These will come in three flavors:
+
+- Informational: Marked with a blue "i" icon on the right side
+- Warning: Marked with an orange exclamation mark icon on the right side
+- Alert: Marked with a red exclamation mark icon on the right side
+
+For example, if company X wants to deprecate the use of one table, they can opt to add a new notice in their configuration file:
```

Comment:
how does this config change for table / dashboard / etc? Possibly: the key be the object URI?

Reply:
Yeah, we currently have different configs for the different resources, so we will be adding a "notices" key in each one of those.

Reply:
which config key in `AppConfig` is for each of them? I'm also OK with deferring this discussion to the PR, that would be fine

Reply:
is a dynamic one [ResourceType.table], [ResourceType.dashboard] and so on